ckanext/geocat/tests/test_dataset_metadata.py

```diff
@@ -205,9 +205,15 @@ def test_fields_values(self):
         self.assertEquals('', dataset.get('accrual_periodicity'))
 
         # see alsos
-        self.assertTrue(hasattr(dataset['see_alsos'], '__iter__'))
-        self.assertEquals(1, len(dataset['see_alsos']))
-        self.assertEquals('8454f7d9-e3f2-4cc7-be6d-a82196660ccd@swisstopo', dataset['see_alsos'][0])  # noqa
+        self.assertTrue(hasattr(dataset['qualified_relations'], '__iter__'))
+        self.assertEquals(1, len(dataset['qualified_relations']))
```

Comment:
here I am not sure, don't we expect that that `qualified_relations` will be a list of dicts?

Reply:
yes, we do, and that's what's being tested here. l209: check that the length of `dataset['qualified_relations']` is 1. l210: check that the first member of `dataset['qualified_relations']` is a dict with the right structure.